README.md

```diff
@@ -1,23 +1,78 @@
+# Generic Torquer – NOS3 Component
+This repository contains the NOS3 Generic Torquer Component. This includes flight software (fsw), ground software (gsw), and simulation (sim) directories.
 
-# Generic_torquer - NOS3 cFS Application
+## Overview
+Torquers, or magnetorquers, are commonly used on spacecraft as a secondary attitude control mechanism. They consist of electromagnets placed asymmetrically. Different currents running through them then generate different magnetic fields, and the torque created by the interaction of the spacecraft magnetic field and the Earth’s magnetic field rotates the vehicle. Because of the small amounts of torque available, magnetorquers are only infrequently used as the primary attitude control mechanism; they frequently serve as a secondary mechanism, though, and are used to provide small rotational changes or to gradually unload reaction wheels. 
+The generic torquer is a device which accepts as input a current (as a percent of the maximum) and returns a value of the generated magnetic field. 
+The available flight software is for use in the core Flight System (cFS) while the ground software supports COSMOS.
+A NOS3 simulation is available which uses a 42 data provider.
 
-This repository contains the NOS3 Generic_torquer cFS application.
+## Technical Specifications
+- Maximum magnetic field strength __
 
-## Overview
-The example assumes a UART based device that.  This example can be used
-with the generic_torquer simulator to show a device that is either synchronous or
-asynchronous (i.e. can stream data).  The generic_torquer shows how to maintain
-and report application status, how to maintain and report device status,
-and how to automatically or on command send device data.  When showing
-device streaming, the streaming rate can be commanded.
-
-## Documentation
-Currently support exists for the following versions:
-* [cFS 6.7/6.8](https://cfs.gsfc.nasa.gov/)
-
-# Commanding
-Refer to the file `fsw/platform_inc/generic_torquer_app_msgids.h` for the Generic_torquer app message IDs
-Refer to the file `fsw/src/generic_torquer_app_msg.h` for the Generic_torquer app command codes
-
-## Versioning
-We use [SemVer](http://semver.org/) for versioning. For the versions available, see the tags on this repository.
+## Mechanical
+### Reference System
+The +z axis is down the length of the magnetorquer (in the same direction as the coil). The +x and +y axes are perpendicular to the coil, and together with the +z axis make an orthogonal, right-handed coordinate system.
+
+## Communications
+The NOS3 Generic Torquer communicates by means of the hardware library (hwlib) defined in https://github.com/nasa-itc/hwlib/tree/dev. Specifically, it makes use of two functions from the torquer library at https://github.com/nasa-itc/hwlib/blob/dev/fsw/public_inc/libtrq.h to send commands to the torquers. These commands wrap the specific functionality of an FPGA or other interface which is used to send a PWM signal.
```

Comment:
Could we swap to using markdown links? E.g.:  
defined in [hwlib](https://github.com/nasa-itc/hwlib/tree/dev).